packages/truffle-codec-utils/src/contexts.ts

```diff
@@ -12,7 +12,7 @@ import { CompilerVersion } from "./compiler";
 
 export namespace Contexts {
 
-  export type Contexts = DecoderContexts | DebuggerContexts;
+  export type Contexts = DecoderContexts | DebuggerContexts | DecoderContextsById;
```

Comment:
We don't need to add this to `Context` also? (I think I understand why not, but just verifying)

Reply:
Correct.